cfgen/cfgen

```diff
@@ -460,6 +463,9 @@ def validate_nodes_desc(nodes_desc: List[Dict[str, Any]]) -> None:
         _assert(node['facts'][ipaddr_key(iface)],
                 f"""{name}: {iface!r} interface has no IP address
 Make sure the value of data_iface in the CDF is correct.""")
+        m0d = node['m0_servers']
+        if m0d is None:
```

Comment:
The same issue with square brackets and KeyError.

Reply:
Done!